File: snippets/fsharp/System.Collections.Generic/DictionaryTKey,TValue/Overview/source2.fs
module source2

open System.Collections.Generic

// Create a new dictionary of strings, with string keys.
let myDictionary = Dictionary<string, string>()

// Add some elements to the dictionary. There are no
// duplicate keys, but some of the values are duplicates.
myDictionary.Add("txt", "notepad.exe")
myDictionary.Add("bmp", "paint.exe")
myDictionary.Add("dib", "paint.exe")
myDictionary.Add("rtf", "wordpad.exe")

//<Snippet11>
for kvp in myDictionary do
    printfn $"Key = {kvp.Key}, Value = {kvp.Value}"
//</Snippet11>

File: snippets/csharp/System.Collections.Generic/DictionaryTKey,TValue/Overview/source2.cs
using System;
using System.Collections.Generic;

public class DictionaryOverviewExample2
{
    public static void Run()
    {
        // Create a new dictionary of strings, with string keys.
        //
        Dictionary<string, string> myDictionary =
            new()
            {
                // Add some elements to the dictionary. There are no
                // duplicate keys, but some of the values are duplicates.
                { "txt", "notepad.exe" },
                { "bmp", "paint.exe" },
                { "dib", "paint.exe" },
                { "rtf", "wordpad.exe" }
            };

        //<Snippet11>
        foreach (KeyValuePair<string, string> kvp in myDictionary)
        {
            Console.WriteLine("Key = {0}, Value = {1}", kvp.Key, kvp.Value);
        }
        //</Snippet11>
    }
}
